modules/react-components/src/components/page-header/page-header.tsx

```diff
@@ -152,12 +157,20 @@ export const PageHeader: React.FunctionComponent<PageHeaderPropsInterface> = (
     );
 
     const headerContentClasses = classNames(
-        "page-header ellipsis",
+        "page-header",
         {
-            "no-max-width": !pageHeaderMaxWidth
+            "no-max-width": !pageHeaderMaxWidth,
+            "truncate-header-content ellipsis": truncateContent
```

Comment:
I think we can just use `ellipsis` herper here and get rid of the custom css class.
```suggestion
            "ellipsis": truncateContent
```

Reply:
Fixed